src/main/kotlin/io/libp2p/core/dsl/Builders.kt

```diff
@@ -133,7 +133,11 @@ open class Builder {
         val secureChannels = secureChannels.values.map { it(privKey) }
         val muxers = muxers.values.map { it() }
 
-        muxers.mapNotNull { it as? StreamMuxerDebug }.forEach { it.muxFramesDebugHandler = debug.muxFramesHandler.handler }
+        val streamVisitors = StreamVisitor.createBroadcast()
```

Comment:
Should this `Builder` have a method to add `StreamVisitor`'s ?

Reply:
This is addressed in the subsequent PR in `DebugBuilder`